admin/sql/updates/alter_last_login.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
```

Comment:
in acousticbrainz we put the date in the filenames of update scripts, to make a it a little easier to know in what order to run things. Should we do this here too?